ecosystem/sep-0010.md

```diff
@@ -23,7 +23,7 @@ The authentication flow is as follows:
 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
 1. The client verifies that the transaction is signed by the `SIGNING_KEY` specified by the requested service's [SEP-1 stellar.toml](sep-0001.md).
-1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client ignores the home domain included.
+1. The client verifies that the transaction's first Manage Data operation has its source account set to the user's account and value set to a nonce value. The client ignores the home domain included.
```

Comment:
I don't want to be nitpicking, but it seems that this wording would allow, for example, a challenge with some `payment` as its first operation, followed by the correct `manage_data` op, and I don't think we should allow that. I guess we need something like "the transaction's first operation is Manage Data..." instead of "transaction's first Manage Data operation".

Reply:
Good point. Addressed in 2efd3b960d7e66b53c7555cd9dd910bb095c73d3.